hack/make/validate-test

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Make sure we're not using gos' Testing package any more in integration-cli
+
+source "$(dirname "$BASH_SOURCE")/.validate"
+
+IFS=$'\n'
+files=( $(validate_diff --diff-filter=ACMR --name-only -- '*.go' | grep '^integration-cli/' || true) )
+unset IFS
+
+badFiles=()
+for f in "${files[@]}"; do
+	# we use "git show" here to validate that what's committed is formatted
+	if [ "$f" = "integration-cli/check_test.go" ]; then
+		continue
+	fi
+
+	if [ "$(git show "$VALIDATE_HEAD:$f" | grep testing.T)" ]; then
```

Comment:
This could be simplified as `if git show "$VALIDATE_HEAD:$f" | grep -q testing.T; then`.